### PR TITLE
restyled download btn

### LINKED ISSUE #4: Restyle "download button" to make it more user-friendly
Currently, the button is flat and awkward. Ideally should be restyled to make it easier for users to understand that it is a button.

### PR DESCRIPTION
**Pull request by darkshredder**
#4 
Without hover:
Screenshot from 2020-10-01 06-40-13
With Hover:
Screenshot from 2020-10-01 06-40-15

### CODE REVIEW

**Comment by darkshredder** [comment]
@arbolitoloco Please review it.

File: index.html [hunk 1]
```diff
@@ -108,7 +108,7 @@ <h2><span class="emphasized">Step 1.</span> Select an image file</h2>
     </div>
     <div id="download-button">
       <h2><span class="emphasized">Step 2.</span> Click button to download gallery</h2>
-      <button onclick="downloadGallery()">Download simulations</button>
+      <button id="download-simulations-btn" onclick="downloadGallery()">Download simulations</button>
```
Comment (arbolitoloco) [inline index.html]: Personally I'd remove the id here.

File: css/style.css [hunk 2]
```diff
@@ -99,6 +99,15 @@ main {
...
 
+#download-simulations-btn {
+  box-shadow: 0 0 20px #ccc;
+  border-radius: 15px;
+  opacity: 0.6;
+}
+#download-simulations-btn:hover {
+  opacity: 1;
+}
+
```
Comment (arbolitoloco) [inline css/style.css:110]: Please note that this button is already inside a `#download-button` div, so I'd call it like this instead:

`#download-button button {..}`
`#download-button button:hover {..}`

I hope it makes sense to you.

Additionally: when NOT on hover, the button seems to have a white overlay on top of it that mutes the button a little. Do you mind correcting the initial state to something brighter? Feel free to change button color as well (you can look at the `button` tag in the `css` to see where the yellow color is coming from.

**Comment by darkshredder** [comment]
Without Hover
Screenshot from 2020-10-01 07-31-06
With Hover
Screenshot from 2020-10-01 07-31-08

**Comment by darkshredder** [comment]
@arbolitoloco does it look better now ?

**Comment by arbolitoloco** [comment]
@darkshredder Much better, thank you!

**Pull request merged**